apps/prairielearn/elements/pl-string-input/pl-string-input.mustache

```diff
@@ -60,6 +87,11 @@
             {{/parse_error}}
         </span>
     </span>
+{{#multiline}}
+    {{#suffix}}
+    <label id="pl-string-input-{{uuid}}-suffix" class="form-text">{{{suffix}}}</label>
```

Comment:
I would not use a `<label>` for the suffix; there's no benefit to that. A `<div>` would probably be more appropriate.

Reply:
https://github.com/PrairieLearn/PrairieLearn/pull/9438/commits/36f38d31ba9446844154c999c4e062636e9f65a0